src/test/skript/tests/regressions/5102-spawn-multiple-entitytypes.sk

```diff
@@ -0,0 +1,7 @@
+test "spawn multiple entities" when running minecraft "1.19":
+	clear all entities
```

Comment:
This shouldn't be needed since there should be a clean state after every test.

Reply:
It wasn't

Reply:
Then you should make sure that's the case by fixing those tests or you specify the entity type in your test to one that hasn't been used.

Reply:
Not the focus of this pull request.